cmd/proxy/actions/app.go

```diff
@@ -132,6 +133,9 @@ func App(conf *config.Config) (http.Handler, error) {
 		lggr,
 		conf,
 	); err != nil {
+		if errors.IsConfigErr(err) {
```

Comment:
so all that we're doing here is choosing *not* to wrap the error with "error adding proxy routes (message)". I think, instead of doing that, we can just pick a better wrapper-message and then we don't need this exposed API at all, we can just say: `"error initializing Athens: %s"` below.